docs/user/power/install_upi.md

```diff
@@ -0,0 +1,298 @@
+
+An example install config for bare-metal UPI is as follows:
+
+```yaml
+apiVersion: v1
+## The base domain of the cluster. All DNS records will be sub-domains of this base and will also include the cluster name.
+baseDomain: example.com
+compute:
+- name: worker
+  replicas: 1
+controlPlane:
+  name: master
+  replicas: 3
+metadata:
+  ## The name for the cluster
+  name: test
+platform:
+  none: {}
+## The pull secret that provides components in the cluster access to images for OpenShift components.
+pullSecret: ''
+## The default SSH key that will be programmed for `core` user.
+sshKey: ''
+```
+
+Create a directory that will be used by the OpenShift installer to provide all the assets. For example `test-bare-metal`,
+
+```console
+$ mkdir test-bare-metal
+$ tree test-bare-metal
+test-bare-metal
+
+0 directories, 0 files
+```
+
+Copy *your* `install-config` to the `INSTALL_DIR`. For example using the `test-bare-metal` as our `INSTALL_DIR`,
+
+```console
+$ cp <your-instal-config> test-bare-metal/install-config.yaml
+$ tree test-bare-metal
+test-bare-metal
+└── install-config.yaml
+
+0 directories, 1 file
+```
+
+NOTE: The filename for `install-config` in the `INSTALL_DIR` must be `install-config.yaml`
+
+### Invoking the installer to get Ignition configs
+
+Given that you have setup the `INSTALL_DIR` with the appropriate `install-config`, you can create the Ignition configs by using the `create ignition-configs` target. For example,
+
+```console
+$ openshift-install --dir test-bare-metal create ignition-configs
+INFO Consuming "Install Config" from target directory
+$ tree test-bare-metal
+test-bare-metal
+├── auth
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+└── worker.ign
+
+1 directory, 4 files
+```
+
+The `bootstrap.ign`, `master.ign`, and `worker.ign` files must be made available as http/https file downloads resolvable by the RHCOS nodes.
+
+## Booting machines with RHCOS and Ignition configs
+
+### Required kernel parameters for IPL boot
+A kernel parameter file must be created for each node with the following parameters:
+
+* `rd.neednet=1`: [CoreOS Installer][coreos-installer] needs internet access to fetch the OS image that needs to be installed on the machine.
+
+* IP configuration [arguments](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-configuring_ip_networking_from_the_kernel_command_line) may be required to access the network.
+
+* CoreOS Installer [arguments][coreos-installer-args] are required to be configured to install RHCOS and setup the Ignition config file for that machine.
+
+* Refer to the following docs for details on booting a Power machine
+  - [iso boot](https://www.ibm.com/developerworks/community/wikis/home?lang=en#!/wiki/Power+Systems/page/Mounting+an+ISO+image+on+VIO+client+LPAR)
+  - [network boot](https://www.ibm.com/developerworks/community/wikis/home?lang=en#!/wiki/Power+Systems/page/How+to+initiate+network+boot+of+an+LPAR)
```

Comment:
Should this be noted that it is PowerVM only and there are different steps if you are putting Linux on bare metal

Reply:
Good point.. Regular baremetal on Power is similar to Intel (PXE boot or via ISO). Only for PowerVM few things are different and hence the links. Is it fine if I change the title of the link from "Refer to the following docs for details on booting a Power machine" to "Refer to the following docs for details on booting a PowerVM machine" ? Or you prefer removing the section altogether ?

Reply:
Made it explicit for PowerVM. Hope it's fine.